iOS_Labs/Views/SignUp/SignUp.swift

```diff
@@ -10,7 +10,11 @@ import SwiftUI
 
 struct SignUp: View {
     @ObservedObject var signUp = SignUpViewModel()
-    @State var errors = [SignUpFieldType :String]()
+    @State var errors = [SignUpFieldType: String]()
+    
+    init() {
```

Comment:
This `init()` should be redundant as we've already setting `signUp` view model in variable initialisation on line 12. Dose it work without such duplication?